src/main.js

```diff
@@ -153,3 +155,27 @@ function showMainPage() {
   makePosterPage.classList.add('hidden');
   savedPostersPage.classList.add('hidden');
 }
+
+function renderCurrentPoster(image, title, quote) {
+  currentPoster = new Poster(image, title, quote);
+
+  posterImage.src = currentPoster.imageURL;
+  posterTitle.innerText = currentPoster.title;
+  posterQuote.innerText = currentPoster.quote; 
+}
+
+function generateOwnPoster(event) {
+  event.preventDefault()
+  
+  renderCurrentPoster(inputImage.value, inputTitle.value, inputQuote.value);
+  saveFormData();
+  showMainPage();
+}
+
+function saveFormData() {
+  var posterElements = [images, titles, quotes]
+  var formElements = [inputImage, inputTitle, inputQuote]
```

Comment:
I would make these variable names a bit more descriptive so that other developers have an easier time understanding exactly what is going on. EX: `userInputFields`